File: src/main/java/net/eduard/permissions/command/PermissionsGroupSetDefaultCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsGroup;
import net.eduard.permissions.manager.PermissionsManager;

public class PermissionsGroupSetDefaultCommand extends CommandManager {

	public PermissionsGroupSetDefaultCommand() {
		super("setdefault","definirpadrao");
		setUsage("/permissions group setdefault <group>");
		setDescription("Definir o grupo padrão do sistema");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 3) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			PermissionsGroup group = manager.getGroup(nome);
			if (group != null) {
				manager.setGroupDefault(group);
				sender.sendMessage(EduPermissions.getInstance().message("group-set-default").replace("$group", "" + nome));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("group-not-exists").replace("$group", "" + nome));
			}
//			
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupSetDefaultCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsGroupSetDefaultCommand :
    Command("setdefault", "definirpadrao") {
    override fun onCommand(
        sender: Sender,
        args: List<String>
    ) {
        val manager = PermissionsAPI.instance
        if (args.size < 3) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        val group = manager.getGroup(nome)
        manager.changeDefaultGroup(group)
        sender.sendMessage(
            PermsMessages.message("group-set-default")
                .replace("\$group", "" + nome)
        )


    }

    init {
        usage = "/permissions group setdefault <group>"
        description = "Definir o grupo padrão do sistema"
    }
}